<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="36501" started-at="2017-08-25T13:20:29Z" finished-at="2017-08-25T13:21:06Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="36501" started-at="2017-08-25T13:20:29Z" finished-at="2017-08-25T13:21:06Z">
      <class name="controller.testsuite.testcase.Func_CollaborateUser">
        <test-method status="PASS" signature="initSeleniumDriver()[pri:0, instance:controller.testsuite.testcase.Func_CollaborateUser@4cf777e8]" name="initSeleniumDriver" is-config="true" duration-ms="3642" started-at="2017-08-25T21:20:29Z" finished-at="2017-08-25T21:20:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initSeleniumDriver -->
        <test-method status="FAIL" signature="verifyDelPersonalInfo()[pri:8, instance:controller.testsuite.testcase.Func_CollaborateUser@4cf777e8]" name="verifyDelPersonalInfo" duration-ms="30767" started-at="2017-08-25T21:20:33Z" finished-at="2017-08-25T21:21:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at processor.proc_CollaborateUser_verifyDelPersonalInfo.init_Search(proc_CollaborateUser_verifyDelPersonalInfo.java:46)
	at controller.testsuite.testcase.Func_CollaborateUser.verifyDelPersonalInfo(Func_CollaborateUser.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyDelPersonalInfo -->
        <test-method status="PASS" signature="deleteSeleniumDriver()[pri:0, instance:controller.testsuite.testcase.Func_CollaborateUser@4cf777e8]" name="deleteSeleniumDriver" is-config="true" duration-ms="2077" started-at="2017-08-25T21:21:04Z" finished-at="2017-08-25T21:21:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteSeleniumDriver -->
      </class> <!-- controller.testsuite.testcase.Func_CollaborateUser -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
